bin/setup

```diff
@@ -11,8 +11,10 @@ def system!(*args)
 end
 
 FileUtils.chdir APP_ROOT do
-  # This script is a way to setup or update your development environment automatically.
-  # This script is idempotent, so that you can run it at anytime and get an expectable outcome.
+  # This script is a way to setup or
+  # update your development environment automatically.
+  # This script is idempotent, so that you can run it
+  # at anytime and get an expectable outcome.
```

Comment:
Looks like a lot of comment line-ending changes. I've adopted the approach of removing all comments from the initial rails generators. If you still feel like you want some documentation, I've linked to whatever relevant page in the rails guides, but often I find myself not even doing that 😄

Reply:
Good point, thank you!